docs/transactions/iti41.md

```diff
@@ -2,40 +2,170 @@
+* [Validity of document](#FIXME) has to be after the [date of the PRE document](#FIXME) and the [start date of duratio](#FIXME)n of treatment of parent MTP 
+* [Code of medication](#FIXME) should match the code of medication of referenced MTP.      
+* If [lot number](#FIXME) is specified in referenced MTP, the PRE item mast match it.
+#### PADV
+##### PADV against a [provisional PRE](#FIXME)
+* Medication cannot be changed.
+* Dosage instructions cannot be changed.
+* Substitution cannot be disallowed if the existing dispense includes a substitution.
+* Observation code `REFUSE` cannot be issued.
+#####  PADV `CANCEL`
+* Not allowed against DIS or other PADV documents.
+
+#### DIS
+* Dispense [status](#FIXME) has to be coherent with previous DIS documents for the same prescription.
+* If [lot number](#FIXME) is specified in parent MTP or PRE, lot number must match.
+##### DIS against PRE:
+* Medication cannot be different if substitution is not allowed by PRE. 
+    * Verification Is performed on the [GTIN](#FIXME) of the medication (not of the package). 
+    * If a substitution other than `none` is specified, no validation is performed.
+* Dispense cannot occur after the [end of validity of the PRE](#FIXME) document.
+* Dispense cannot occur after the [end of the treatment](#FIXME) if specified in the [prescription item](#FIXME).
+* If referenced PRE is not [provisional](#FIXME), it should be [dispensable](#FIXME).
+* List of active ingredients should be the same than the one in the [parent PRE](#FIXME).
+* If substitution occurs, the [medication code (GTIN)](#FIXME) should be the same than the medication code from the [parent PRE](#FIXME).
+##### Against MTP:
+* Medication cannot be different if [substitution](#FIXME) is not allowed by the MTP item. Verification is done on medication's [GTIN](#FIXME).
+* Dispense cannot occur after the [end of the treatment](#FIXME) if specified in the MTP item.
+* Referenced MTP must be [approved](#FIXME) and [active](#FIXME) and for the same patient.
+* [List of active ingredients](#FIXME) should be the same than the one in the [parent MTP](#FIXME).
+* If substitution occurs, the [medication code (GTIN)](#FIXME) should be the same than the medication code from the [parent MTP](#FIXME).
+### Roles
+* [Patients](#FIXME) and their [representatives](#FIXME) are allowed to publish only the following type of docs:
+    * MTP
+    * DIS
+    * PADV with a COMMENT observation code with no restriction.
+    * PADV with any other observation code only when referencing a document published by the same patient or a representative of the same patient.     
+* [Healthcare providers](#FIXME) are allowed to publish only the following types of docs:
+    * MTP
+    * PRE
+    * DIS
+    * PADV		
+* [Creation time of the document](#FIXME) must be equal or greater than the creation time of the last document of the [treatment chain](#FIXME).
```

Comment:
- Creation time of the document: `CHEMEDDocumentMedicationPrescription.Composition.date` and/or `CHEMEDDocumentMedicationPrescription.timestamp`.
- The last document of the treatment chain is the last published document targeting the treatment (either the original MTP if no other document, or the last document published referencing this treatment).